src/util/accessibility/live.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable, Inject, OnDestroy} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class Live implements OnDestroy {
+  private _element: HTMLElement;
+
+  constructor(@Inject(DOCUMENT) private _document: any) { this._element = this._createElement(); }
+
+  ngOnDestroy() { this._element.remove(); }
+
+  async say(message: string): Promise<void> {
```

Comment:
Could you please not use async-await for now? It transpiles to rather verbose code.

Reply:
More important - why do we return a `Promise` here? We don't care about any return value, do we? What would be the use-case for knowing when announcing starts?

Reply:
I think this is pity to deprive ourselves of one of the best __standard__ language features that was brought these last years: now asynchronous code remains explicit and controlled, but is aligned with synchronous code. It's what made me adopt async code widely, because otherwise dealing with Promises chaining is quite verbose.

This is a general comment, and of course here using a Promise is easy, since I even had to create one due to the lack of proper API for creating a delay.

However: 

- first, the code size shouldn't be an issue, especially once compressed
- then, we should configure the compilation so that it doesn't generate the helpers in each file, but instead relies on the TypeScript library to have this piece of code only once.
- last, we don't really care about the return value here. It's a `Promise` because async functions return Promises.